doc/JSON_INFO.md

```diff
@@ -2492,6 +2492,13 @@ Alternately, every item (book, tool, armor, even food) can be used as a gunmod i
 "use_action": [ "firestarter" ] // Action performed when tool is used, see special definition below
 ```
 
+Items that define any of these fields must be declared as type "TOOL" in order to work correctly:
+ammo, max_charges, initial_charges, charges_per_use, charge_factor, power_draw, revert_to, revert_to_msg,
+sub, rand_charges.
+
+If an item doesn't have any of those fields, it should not be type "TOOL".
```

Comment:
Minor, but I'd probably put this right under the Tools section header on L2464 since it defines when and when not to use it.